packages/instant/src/index.umd.ts

```diff
@@ -122,6 +128,43 @@ export const render = (config: ZeroExInstantConfig, selector: string = DEFAULT_Z
     window.onpopstate = onPopStateHandler;
 };
 
+export const assetDataForERC20TokenAddress = (tokenAddress: string): string => {
+    assert.isETHAddressHex('tokenAddress', tokenAddress);
+    return assetDataUtils.encodeERC20AssetData(tokenAddress);
+};
+
+export const hasMetaDataForAssetData = (assetData: string): boolean => {
+    assert.isHexString('assetData', assetData);
+    return assetMetaDataMap[assetData] !== undefined;
+};
+
+export const hasLiquidityForAssetDataAsync = async (
+    assetData: string,
+    orderSource: OrderSource,
+    networkId: Network = Network.Mainnet,
+    provider?: Provider,
+): Promise<boolean> => {
+    assert.isHexString('assetData', assetData);
+    assert.isValidOrderSource('orderSource', orderSource);
+    assert.isNumber('networkId', networkId);
+
+    if (provider !== undefined) {
+        assert.isWeb3Provider('provider', provider);
+    }
+
+    const bestProvider: Provider = provider || providerFactory.getFallbackNoSigningProvider(networkId);
+
+    const assetBuyerOptions = { networkId };
+
+    const assetBuyer =
+        typeof orderSource === 'string'
```

Comment:
hmm we should use `_.isString(orderSource` here because it is consistent with how we are type checking in `asset_buyer_factory.ts`, or change it there